test/common_utils.py

```diff
@@ -492,6 +492,34 @@ def get_num_channels(color_space):
     return num_channels
 
 
+def make_image(
+    spatial_size,
+    *,
+    color_space="RGB",
+    batch_dims=(),
+    dtype=torch.float32,
+    device="cpu",
+    constant_alpha=True,
```

Comment:
Do we ever use that (in the new tests)?

Reply:
So far, we have been only testing on RGB and this is a setting that only applies to `"GRAYSCALE_ALPHA"` and `"RGBA"`. I remember we put this in for `convert_color_space`, but since we ripped that out later, not sure that we still need it. IMO it depends if we want to keep the generic `color_space` parameter. If so, I would keep that as well or at least hardcode the behavior, to avoid producing unrealistic images.

Reply:
As discussed offline, we'll remove this parameter for now.